.github/pull_request_template.md

```diff
@@ -1,7 +1,9 @@
 <!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
 
 ## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
+In case you are a technological partner:
 - [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.
+If you are not a technological partner (i.e. PANW employee, customer or individual developer), please delete this section.
```

Comment:
I think you wanted this on a new line. It is showing on the same line as there is no space after the . in the line above.